### ISSUE TITLE
IDC-MCX

### ISSUE THREAD

**Issue by jordens** [sinara-hw/meta#70]
This is a proposal for a very simple new break-out board: analog 2xIDC to 16xMCX.
Electrically similar to IDC-BNC but mechanics similar to [DIO-MCX](https://github.com/sinara-hw/DIO_MCX/).
For Fastino and Zotino to enable higher density setups.
Whether the resulting cable density is desirable is another question.

**Comment by gkasprow** [comment]
yes, connecting all 32 channels of Zotino to BNCs consumes a lot of panel space.

**Comment by gkasprow** [comment]
the easiest way would be to use a single PCB and vertical MCX connector. But the EMC lips and RC filters won't be possible here
![obraz](https://user-images.githubusercontent.com/4325054/140104145-cf6d28ca-e7c1-4378-a4e4-716012997a70.png)
Another approach is dual-level PCB like in DIO-MCX, slightly more expensive but without compromises

**Comment by jordens** [comment]
Cool idea. I don't think we really need the CMC and RC filters on there. But the EMC lip would be desirable.
If you do a single 8 HP (or 12 HP) panel for 32 channels, can it be made to work with a single PCB?

**Comment by gkasprow** [comment]
32 channels should fit on 8HP with EMC lip, but it could be difficult to unplug some connectors selectively using standard fingers :)

**Comment by gkasprow** [comment]
that could work
![obraz](https://user-images.githubusercontent.com/4325054/140163679-01f369e9-095f-4ab4-9640-d2aaca3f13c0.png)
![obraz](https://user-images.githubusercontent.com/4325054/140163720-92c01678-e374-4992-bfa9-53b86633f1c7.png)

**Comment by gkasprow** [comment]
to include the CMCs we would need to go for 12HP or dual PCB approach

**Comment by gkasprow** [comment]
on the other side, what is the benefit of having a 12HP solution since 4x SMA gives 16HP...

**Comment by gkasprow** [comment]
we can also think of an 8HP mezzanine sitting on top of Zotin/Fastino which could work without IDC cables at all.

**Comment by jordens** [comment]
Let's leave the CMC s out. We removed them from fasting and from the other IDC boards already.
If you can.make it work without the IDC cables, even better. I lack the Imagination what that would look like with a single PCB. ;)

**Comment by gkasprow** [comment]
well, I didn't say it would be a single PCB :)

**Comment by gkasprow** [comment]
This is the idea of the Zotino/Fastino mezzanine
![obraz](https://user-images.githubusercontent.com/4325054/140652221-e04607da-30c4-400e-824f-7457ef4b73a5.png)

One can also solder in the IDC connectors and CMC filters and use it stand-alone
![obraz](https://user-images.githubusercontent.com/4325054/140652240-dcd9db17-df1d-4d2f-8a7d-0f2f343c891f.png)

panel
![obraz](https://user-images.githubusercontent.com/4325054/140652996-9c9dcce9-f33b-4ffe-9011-8bd07c6735ed.png)
